index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="fr">
+    <!--head-->
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta
+            name="description"
+            content="Ohmyfood des menus gastronomiques à commander en ligne pour ne plus devoir attendre au restaurant"
+        />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Oh my food</title>
+        <link rel="preconnect" href="https://fonts.gstatic.com" />
+        <link
+        href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&family=Shrikhand&display=swap"
+        rel="stylesheet"
+        />
```

Comment:
You have to remove it, it's on typography

Reply:
i removed this link because it is in typography.scss